### ISSUE TITLE
Hide bar when using Max layout? 

### ISSUE THREAD

**Issue by tamis-laan** [qtile/qtile#1787]
When using the max layout to go to full screen I would like to hide the bar. The only way to hide the bar as far as I could find is the cmd_hide_show_bar(). However this only toggles the bar, one would need to explicitly hide or show hooking into the layout_change group_change. Any way to make this work?

**Comment by tamis-laan** [comment]
Managed to hack it together:
```python 
def main(qtile):
    # Show/Hide bar based on layout
    def _bar(qtile):
        # Get the bar 
        bar = qtile.current_screen.bottom
        # Check the layout and hide bar accordingly
        if(qtile.current_layout.info()['name'] == 'max'):
            bar.show(False)
        else:
            bar.show(True)

    @hook.subscribe.layout_change
    def layout_change(layout,group):
        _bar(qtile)

    @hook.subscribe.changegroup
    def group_change():
        _bar(qtile)

    @hook.subscribe.client_focus
    def focus_change(window):
        _bar(qtile)
```
However getting strange behavior where the bar content is hidden but the bar window itself remains.

**Comment by tamis-laan** [comment]
It seams as if:
```python
    @hook.subscribe.changegroup
    def group_change():
        _bar(qtile)
```
is never triggered. Could it be that changegroup hook is bugged?

**Comment by tych0** [comment]
On Sat, Jun 20, 2020 at 01:30:43AM -0700, Tamis wrote:
> It seams as if:
> ```python
>     @hook.subscribe.changegroup
>     def group_change():
>         _bar(qtile)
> ```
> is never triggered. Could it be that changegroup hook is bugged?

Possibly, I think someone mentioned this before.

**Comment by tamis-laan** [comment]
I'm having doubts, seams as if the hook is triggered however the bar is not being drawn causing the bar to behave like a gap. By changing the layout back and forth i.e. `layout_change` the bar is properly drawn again. For me not very easy to debug.

**Comment by ramnes** [comment]
Fixed in #1926
